lib/uri.js

```diff
@@ -90,6 +93,10 @@ URI.prototype.toString = function(options) {
                 // Omit optional segment & return
                 return uriStr;
             }
+        } else if (this._pathMetadata
+                && this._pathMetadata[i]
+                && this._pathMetadata[i] === '+') {
+            uriStr += '/' + utils.encodeReserved(segment);
```

Comment:
@gwicke No, we can't skip it. The reserved set is described [here](https://github.com/wikimedia/swagger-router/pull/25/files#diff-50e3aa130a4f97a42ee2cf111c7b1d9dR78) - it's not all the characters. [Here's](http://tools.ietf.org/html/rfc6570#page-23) an example: "Hello world!" is expanded to "Hello%20World!" - most of the characters are still encoded.